upgrade/1.4.4/README.md

```diff
@@ -0,0 +1,382 @@
+# CSM 1.4.4 Patch Installation Instructions
+
+* [Introduction](#introduction)
+* [Bug fixes and improvements](#bug-fixes-and-improvements)
+* [Steps](#steps)
+
+## Introduction
+
+This document guides an administrator through the patch update to Cray Systems Management `v1.4.4` from `v1.4.0`, `v1.4.1`,
+`1.4.2`, or `1.4.3`. If upgrading from CSM `v1.3.x` directly to `v1.4.4`, follow the procedures described
+in [Upgrade CSM](../README.md) instead.
+
+## Bug fixes and improvements
```

Comment:
If this section is blank, should it not just be omitted?

Reply:
This comment wasn't addressed in your earlier pass. Not sure if it was overlooked.

Reply:
This is special.
I don't know what to put here, normally a manager puts things here but all of our managers have left us.

Reply:
I can add notes for the marvell thing, and for whatever else is in the branch https://cray.slack.com/archives/C06F199JR4N/p1706058666633919

Reply:
Updated.

@haasken-hpe , @spillerc-hpe , @rfrost-hpe , and @mitcharf please inspect the "Bug fixes and improvements"

Reply:
The SAT entry is accurate.